spec/datadog/tracing/distributed/helpers_spec.rb

```diff
@@ -73,12 +73,10 @@
     context 'when value is' do
       [
         [nil, nil],
+        ['not a number', nil],
+        ['1.to_i returns 1', nil],
```

Comment:
Minor: I'm not entirely sure it's clear what we're testing here (and to be fair, I think this was part of the problem of the old test -- it was actually not clear what it was testing).

I suggest perhaps `['1 2 3', nil], # '1 2 3'.to_i => 1 and we have extra logic to avoid this`.

(Also applies below)